src/Bridges/Nette/TracyExtension.php

```diff
@@ -19,6 +19,8 @@ class TracyExtension extends Nette\DI\CompilerExtension
 {
 	public $defaults = array(
 		'email' => NULL,
+		'logDirectory' => NULL,
+		'logSeverity' => 0,
```

Comment:
I know nothing about writing extension and thought that the value should correspond to default value of `Debugger::$logSeverity`.